src/main/sensors/acceleration.c

```diff
@@ -70,14 +70,21 @@ static uint16_t calibratingA = 0;      // the calibration is done is the main lo
 
 static biquadFilter_t accFilter[XYZ_AXIS_COUNT];
 
+#ifdef USE_ACC_NOTCH
+static filterApplyFnPtr accNotchFilterApplyFn;
+static void *accNotchFilter[XYZ_AXIS_COUNT];
+#endif
+
 PG_REGISTER_WITH_RESET_FN(accelerometerConfig_t, accelerometerConfig, PG_ACCELEROMETER_CONFIG, 0);
```

Comment:
Please increase PG version (replace `0` with `1`) to enforce accelerometer parameter reset.